netlify/functions/handler.js

```diff
@@ -0,0 +1,50 @@
+const { graphql } = require('@octokit/graphql')
+const repositoryID = "MDEwOlJlcG9zaXRvcnkzNDc2MjE1NTk="
+const categoryID = "DIC_kwDOFLhIt84B_T4d"
+
+exports.handler = async function (event, context) {
```

Comment:
I don't think we use `context` here

Reply:
Yeah, I forgot to remove it from the function. I will remove it in the next commit.